src/azul/__init__.py

```diff
@@ -1565,6 +1564,13 @@ def indexer_queue_names(self) -> list[str]:
             for retry in (False, True)
         ]
 
+    @property
+    def work_fail_queue_names(self) -> list[str]:
```

Comment:
A queue is either a work queue or a fail queue so this name makes no sense.

```suggestion
    def indexer_fail_queue_names(self) -> list[str]:
```